scripts/build_all.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import argparse
+import os
+import subprocess
+
+VERSION = '1.13.1'
+REPO = 'sagemaker-tensorflow-scriptmode'
+PY2_CPU_BINARY = 'https://s3-us-west-2.amazonaws.com/tensorflow-aws/1.13/AmazonLinux/cpu/latest-patch-latest-patch/tensorflow-1.13.1-cp27-cp27mu-linux_x86_64.whl'
+PY3_CPU_BINARY = 'https://s3-us-west-2.amazonaws.com/tensorflow-aws/1.13/AmazonLinux/cpu/latest-patch-latest-patch/tensorflow-1.13.1-cp36-cp36m-linux_x86_64.whl'
+PY2_GPU_BINARY = 'https://s3-us-west-2.amazonaws.com/tensorflow-aws/1.13/AmazonLinux/gpu/latest-patch-latest-patch/tensorflow-1.13.1-cp27-cp27mu-linux_x86_64.whl'
+PY3_GPU_BINARY = 'https://s3-us-west-2.amazonaws.com/tensorflow-aws/1.13/AmazonLinux/gpu/latest-patch-latest-patch/tensorflow-1.13.1-cp36-cp36m-linux_x86_64.whl'
+DEV_ACCOUNT = '142577830533'
+REGION = 'us-west-2'
+
+def _parse_args():
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--account', type=str, default=DEV_ACCOUNT)
+    parser.add_argument('--region', type=str, default=REGION)
+    parser.add_argument('--version', type=str, default=VERSION)
+    parser.add_argument('--py2-cpu-binary', type=str, default=PY2_CPU_BINARY)
+    parser.add_argument('--py3-cpu-binary', type=str, default=PY3_CPU_BINARY)
+    parser.add_argument('--py2-gpu-binary', type=str, default=PY2_GPU_BINARY)
+    parser.add_argument('--py3-gpu-binary', type=str, default=PY3_GPU_BINARY)
+    parser.add_argument('--repo', type=str, default=REPO)
+
+    return parser.parse_args()
+
+
+args = _parse_args()
+binaries = {
+    'py2-cpu': args.py2_cpu_binary,
+    'py3-cpu': args.py3_cpu_binary,
+    'py2-gpu': args.py2_gpu_binary,
+    'py3-gpu': args.py3_gpu_binary
+}
+build_dir = 'docker/{}'.format(args.version)
+
+# Run docker-login so we can pull the cached image
+login_cmd = subprocess.check_output(
+    'aws ecr get-login --no-include-email --registry-id {}'.format(args.account).split())
+print(login_cmd)
```

Comment:
I'd preface each command with a description of what is being printed out. same goes for ll. 62 and 69 and similar lines in the other file

Reply:
k